email/1.3/modules/ROOT/edits/email-gmail.adoc

```diff
@@ -0,0 +1,160 @@
+
+To connect Email Connector to your Gmail account using a POP3S connection, configure a global element:
+
+. In Studio, navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field, type `email`, select *Email POP3*, and click *OK*.
+. In the *Email POP3* window, for the *Connection* field, select *POP3S Connection*.
+. In the *General* section, enter the following values:
++
+* *Host* +
+`pop.gmail.com`
+* *Port* +
+`993`
+* *User* +
+`user@gmail.com`
+* *Password* +
+`Gmailpassword`
++
+[start=6]
+. In the *Trust Store Configuration* section, select the *Insecure* checkbox.
+. Click *OK*.
+
+The following screenshot shows the configuration:
+
+.POP3S Configuration to Gmail
+image::email-gmail-pop3.png[POP3S Configuration to Gmail]
+
+In the XML editor, the configuration looks like this:
+
+
+[source,xml,linenums]
+----
+<email:pop3-config name="Email_POP3">
+    <email:pop3s-connection host="pop.gmail.com"
+      port="995"  user="user@gmail.com"
+      password="Gmailpassword">
+        <tls:context>
+            <tls:trust-store insecure="true"/>
+        </tls:context>
+    </email:pop3s-connection>
+</email:pop3-config>
+----
+
+== Configure a SMTP Connection
+
+To connect Email Connector to your Gmail account using a SMTPS connection, configure a global element:
+
+. In Studio, navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field, type `email`, select *Email SMTP*, and click *OK*.
+. In the *Email SMTP* window, for the *Connection* field, select *SMTPS Connection*.
+. In the *General* section, enter the following values:
++
+* *Host* +
+`smtp.gmail.com`
+* *Port* +
+`465`
+* *User* +
+`user@gmail.com`
+* *Password* +
+`Gmailpassword`
++
+[start=6]
+. In the *Trust Store Configuration* section, select the *Insecure* checkbox.
+. Click *OK*.
+
+The following screenshot shows the configuration:
+
+.SMTPS Configuration to Gmail
+image::email-gmail-smtp.png[SMTPS Configuration to Gmail]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<email:smtp-config name="Email_SMTP" from="sender@gmail.com">
+    <email:smtps-connection host="smtp.gmail.com"
+      port="465" password="Gmailpassword"
+      user="user@gmail.com">
+        <tls:context enabledProtocols="TLSv1.2,SSLv3">
+            <tls:trust-store insecure="true"/>
+        </tls:context>
+    </email:smtps-connection>
+</email:smtp-config>
+----
+
+== Troubleshoot SMTPS Connection Issues
+
+Troubleshooting SMTPS connection issues depends on whether or not you have two-factor authentication set up for your Gmail account:
+
+If you have two-factor authentication set up, generate an app-specific password and use that instead of your normal password. See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details. There is no need to enable *Less Secure Apps* in your Gmail account.
```

Comment:
```suggestion
* If your account uses two-factor authentication, generate an app-specific password and use that instead of your normal password. 
+
See https://support.google.com/accounts/answer/185833[Sign in Using App Password] for details. You do not need to enable *Less Secure Apps* in your Gmail account.
```

Reply:
(suggested adding this caveat above)